src/sql/src/session/vars.rs

```diff
@@ -2415,6 +2423,9 @@ impl SystemVars {
                 .lock()
                 .expect("lock poisoned")
                 .limit = u64::cast_from(*self.expect_value(&MAX_CONNECTIONS));
+        } else if name == VARIABLE_LENGTH_ROW_ENCODING.name {
+            let value = *self.expect_value(&VARIABLE_LENGTH_ROW_ENCODING);
+            mz_repr::VARIABLE_LENGTH_ROW_ENCODING.store(value, atomic::Ordering::SeqCst);
```

Comment:
What is the rationale for the `Ordering::SeqCst` here? I can't really say I grok the memory orderings, but I know that `SeqCst` is the strictest one and therefore potentially the most expensive one too. Would a less strict ordering suffice here?

Reply:
A less strict ordering is probably fine, however, I chose to go with a stricter one to avoid having to think about the consequences of that. In general, I always choose the strictest ordering unless I have good reason to believe it's a performance bottleneck.

In this case, having the strictest possible ordering shouldn't change performance, because:

(1) on x86, reads are always atomic, so this is a no-op on the load side (it will emit the same instructions as any memory load), and
(2) although it might be slower on the store side, that is okay, since we are only storing into this value on LD changes.

Reply:
Cool, if it doesn't matter for read performance, I agree with your reasoning!